packages/bpk-component-banner-alert/src/BpkBannerAlert.module.scss

```diff
@@ -87,6 +87,7 @@
   }
 
   &__message {
+    margin-top: bpk-spacing-sm() / 2;
```

Comment:
Its an interesting on, as I had a look and it seems that the icon puts its own inline styles which adds an additional margin to the top which means that it can't align correctly.

So it might actually be the icon component itself without this it might be the text is aligned and the icon is the issue here 🤔 
 
![Screenshot 2021-08-12 at 18 31 24](https://user-images.githubusercontent.com/8831547/129242030-2122b2d6-9aa0-4fe5-9c93-1c9368469445.png)

Reply:
Gonna double check as it might be that the icon alignment HoC with BpkText could help here

Reply:
Looks like we just need to put `align-items: center;` on the outer flex container...

Reply:
Oo will try that tomorrow, I also had a go using our alignment hocs for the icon which did the job too! So many choices